corehq/apps/case_importer/tracking/filestorage.py

```diff
@@ -28,12 +28,11 @@ def __init__(self, meta_model):
         :meta_model is a django model used to store meta info
             must contain columns identifier, filename, length
         """
-        self._db = get_blob_db()
         self._meta_model = meta_model
 
     def write_file(self, f, filename, domain):
         identifier = random_url_id(16)
-        meta = self._db.put(
+        meta = get_blob_db().put(
```

Comment:
`PersistentFileStore.__init__()` is called as part of HQ startup (due to imports), and `get_blob_db()` should be very fast after the first time it is called. Had to do this to prevent [assertion failure in `set_max_connections`](https://github.com/dimagi/commcare-hq/pull/22815/files#diff-3e20f7a8a707b849346106c493836410R120).